### DISCUSSION TITLE
How to disable entire category or range of NIDS rules

### DISCUSSION THREAD

**Discussion by DebianGuru** [Security-Onion-Solutions/securityonion#2907] [Unsupported Versions]
In the old version of SO, I could simply disable a whole category of my ET definitions by simply adding a line like "Custom-games" to my disablesid.conf file.

The documentation shows how do disable individual rules by ID and even using Regular expressions. 

Is there a way to disable an entire category within the .sls file?  What is the syntax? What about ranges of rule numbers?  Surely, we cant add the 10,000 rules we want to disable one line at a time??? I'm sure this is a common situation.  
 I'm sorry to ask, but I can't find it in the documentation and I've found nothing on this site.  Any help is appreciated.

**Comment by weslambert** [comment]
Have you tried something like the following (using `classtype`?) 

```
idstools:
  sids:
    disabled:
      - 're:credential-theft'
```
I'm not sure if you can disable by "category" with idstools as with PulledPork.  Speaking to range of numbers, you could just write a regex for the range as wel.
